### PR TITLE
Remove Kube*Overcommit alerts

### PR DESCRIPTION
**Pull request by vulkoingim**
## Description

https://github.com/gitpod-io/ops/pull/6259 enables autoscaling on the only nodepool that doesn't have it still, which renders these alerts obsolete.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by ArthurSens** [comment]
Not sure why we're the owners of the platform folder 🤔 

Would it make sense to change the ownership?

**Review by ArthurSens — APPROVED** [review]
Hey @vulkoingim, I'm approving just because I don't want to block this for too long 😬 

I'm just not sure that the change to CODEOWNERS worked as expected, I would assume that the review-request would have changed 🤔

**Comment by vulkoingim** [comment]
> Hey @vulkoingim, I'm approving just because I don't want to block this for too long 😬
> 
> I'm just not sure that the change to CODEOWNERS worked as expected, I would assume that the review-request would have changed 🤔

Thanks! The CODEOWNERS is read from the main branch - so the change will take effect for all following PRs :) Otherwise anyone can modify the owners file and set themselves as approvers 😄

**Comment by ArthurSens** [comment]
> > Hey @vulkoingim, I'm approving just because I don't want to block this for too long 😬
> > I'm just not sure that the change to CODEOWNERS worked as expected, I would assume that the review-request would have changed 🤔
> 
> Thanks! The CODEOWNERS is read from the main branch - so the change will take effect for all following PRs :) Otherwise anyone can modify the owners file and set themselves as approvers 😄

Oh yes of course

**Pull request merged**
